src/main/java/ch/njol/skript/expressions/ExprVersionString.java

```diff
@@ -37,43 +32,49 @@
 import ch.njol.util.Kleenean;
 import ch.njol.util.coll.CollectionUtils;
+import com.destroystokyo.paper.event.server.PaperServerListPingEvent;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
 
 @Name("Version String")
-@Description({"The text to show if the protocol version of the server doesn't match with protocol version of the client. " +
-		"You can check the <a href='#ExprProtocolVersion'>protocol version</a> expression for more information about this.",
-		"This can only be set in a <a href='events.html#server_list_ping'>server list ping</a> event."})
-@Examples({"on server list ping:",
-		"	set the protocol version to 0 # 13w41a (1.7), so it will show the version string always",
-		"	set the version string to \"&lt;light green&gt;Version: &lt;orange&gt;%minecraft version%\""})
+@Description({
+	"The text to show if the protocol version of the server doesn't match with protocol version of the client. " +
+	"You can check the <a href='#ExprProtocolVersion'>protocol version</a> expression for more information about this.",
+	"This can only be set in a <a href='events.html#server_list_ping'>server list ping</a> event."})
+@Examples({
+	"on server list ping:",
+	"	\tset the protocol version to 0 # 13w41a (1.7), so it will show the version string always",
+	"	\tset the version string to \"&lt;light green&gt;Version: &lt;orange&gt;%minecraft version%\""
+})
 @Since("2.3")
-@RequiredPlugins("Paper 1.12.2 or newer")
-@Events("server list ping")
+@RequiredPlugins("Paper 1.12.2+")
+@Events("Server List Ping")
 public class ExprVersionString extends SimpleExpression<String> {
 
+	private static final boolean PAPER_EVENT_EXISTS = Skript.classExists("com.destroystokyo.paper.event.server.PaperServerListPingEvent");
+
 	static {
-		Skript.registerExpression(ExprVersionString.class, String.class, ExpressionType.SIMPLE, "[the] [(shown|custom)] version [(string|text)]");
+		Skript.registerExpression(ExprVersionString.class, String.class, ExpressionType.SIMPLE, "[the] [shown|custom] version [string|text]");
 	}
 
-	private static final boolean PAPER_EVENT_EXISTS = Skript.classExists("com.destroystokyo.paper.event.server.PaperServerListPingEvent");
-
 	@Override
 	public boolean init(Expression<?>[] exprs, int matchedPattern, Kleenean isDelayed, ParseResult parseResult) {
-		if (getParser().isCurrentEvent(ServerListPingEvent.class)) {
-			Skript.error("The version string expression requires Paper 1.12.2 or newer");
+		if (!PAPER_EVENT_EXISTS) {
+			Skript.error("The 'version string' expression requires Paper 1.12.2+");
 			return false;
-		} else if (!(PAPER_EVENT_EXISTS && getParser().isCurrentEvent(PaperServerListPingEvent.class))) {
-			Skript.error("The version string expression can't be used outside of a server list ping event");
+		} else if (!getParser().isCurrentEvent(PaperServerListPingEvent.class)) {
+			Skript.error("The 'version string' expression can't be used outside of 'server list ping' event");
```

Comment:
```suggestion
			Skript.error("The 'version string' expression can't be used outside of a server list ping event");
```
I think the previous version is better

Reply:
Applied suggested change discussed in skUnity's SkriptLang channel.